src/main/java/seedu/duke/command/task/AddTaskCommand.java

```diff
@@ -47,6 +54,31 @@ public void execute() {
         Ui.showToUserLn("Task successfully created.");
         Ui.showToUserLn(addedTask.toString());
     }
+
+    public boolean doesTitleExist(String title) {
+        boolean titleAlreadyExist = false;
+        if (projectManager.isEmpty()) {
+            return false;
+        }
+        for (int i = 1; i <= projectManager.getSelectedProject().getProjectBacklog().size(); i++) {
```

Comment:
Does this check for matching task description or only the title?

Reply:
Because I think it does not matter, should only check title

Reply:
My implementation currently only checks the title so I think this part's good

Reply:
Sure that works

Reply:
I have tested it works on my side, I believe it is functional